openml/setups/setup.py

```diff
@@ -0,0 +1,35 @@
+
+class OpenMLSetup(object):
+    """Setup object (a.k.a. Configuration).
+
+       Parameters
+       ----------
+       flow_id : int
+            The flow that it is build upon
+        parameters : dict
+            The setting of the parameters
+           """
+
+    def __init__(self, flow_id, parameters):
+        self.flow_id = flow_id
+        self.parameters = parameters
+
+
+class OpenMLParameter(object):
+    """Parameter object (used in setup).
+
+       Parameters
+       ----------
+       flow_id : int
```

Comment:
It looks like you copied the docstring from the object above and didn't adapt it.